egs/gop_speechocean762/s5/run.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
+
+# Copyright      2019  Junbo Zhang
+# Copyright 2020-2021  Xiaomi Corporation (Author: Junbo Zhang, Yongqing Wang)
+# Apache 2.0
+
+# This script shows how to calculate Goodness of Pronunciation (GOP) and
+# use the GOP-based feature to do phone-level mispronunciations detection.
+# Read ../README.md or the following paper for details:
+#
+# "Hu et al., Improved mispronunciation detection with deep neural network
+# trained acoustic models and transfer learning based logistic regression
+# classifiers, 2015."
+
+
+# Set this to somewhere where you want to put your data, or where someone 
+# else has already put it.  You'll want to change this if you're not on
+# the Xiaomi's grid.
+data=/home/storage07/zhangjunbo/data
+
+# base url for downloads.
+data_url=www.openslr.org/resources/93
+stage=1
+nj=25
+
+# You might not want to do this for interactive shells.
+set -e
+
+. ./cmd.sh
+. ./path.sh
+. parse_options.sh
+
+# This recipe depends on the model trained in the librispeech recipe.
+# For training it:
+#   cd $KALDI_ROOT/egs/librispeech/s5
+#   ./run.sh && local/nnet3/run_tdnn.sh
+
+# Check librispeech's models
+librispeech_eg=../../librispeech/s5
+model=$librispeech_eg/exp/nnet3_cleaned/tdnn_sp
+ivector_extractor=$librispeech_eg/exp/nnet3_cleaned/extractor
+lang=$librispeech_eg/data/lang
+
+for d in $model $ivector_extractor $lang; do
+  [ ! -d $d ] && echo "$0: no such path $d" && exit 1;
+done
+
+if [ $stage -le 1 ]; then
+  # Download data and untar
+  local/download_and_untar.sh $data_url $data
+fi
+
+if [ $stage -le 2 ]; then
+  # Prepare data
+  for part in train test; do
+    local/data_prep.sh $data/speechocean762/$part data/$part
+  done
+
+  mkdir -p data/local
+  cp $data/speechocean762/resource/* data/local
+fi
+
+if [ $stage -le 3 ]; then
+  # Create high-resolution MFCC features
+  for part in train test; do
+    steps/make_mfcc.sh --nj $nj --mfcc-config conf/mfcc_hires.conf \
+      --cmd "$cmd" data/$part || exit 1;
+    steps/compute_cmvn_stats.sh data/$part || exit 1;
+    utils/fix_data_dir.sh data/$part
+  done
+fi
+
+if [ $stage -le 4 ]; then
+  # Extract ivector
```

Comment:
No hurry, but it would be better to remove ivector later as it may not help for Pronunciation scoring task

Reply:
Yes, I agree. The ivector extraction in this recipe is only for reusing the librispeech model.